scripts/globals/spells/bio_ii.lua

```diff
@@ -47,8 +48,26 @@ function onSpellCast(caster,target,spell)
     -- Check for Dia
     local dia = target:getStatusEffect(tpz.effect.DIA)
 
-    -- Calculate DoT effect (rough, though fairly accurate)
-    local dotdmg = 3 + math.floor(caster:getSkillLevel(tpz.skill.DARK_MAGIC) / 60)
+    -- Calculate DoT effect. Unknown formula, but known cap of 8 and some known breakpoints.
+    local dotdmg = 0
+    if skillLvl > 290 then
+        dotdmg = 8
+    elseif skillLvl > 268 then
+        dotdmg = 7
+    -- known breakpoints above this line. unknown breakpoints below. for unknown breakpoints, I divided the remaining skill equally
+    elseif skillLvl > 223 then
+        dotdmg = 6
+    elseif skillLvl > 178 then
+        dotdmg = 5
```

Comment:
[old wiki is claiming](https://ffxiclopedia.fandom.com/wiki/Bio_II):
```
    290~ skill: 8 HP/3 seconds
    251~290 skill: 7 HP/3 seconds
    211~250 skill: 6 HP/3 seconds
    171~210 skill: 5 HP/3 seconds
    131~170 skill: 4 HP/3 seconds
    0~130 skill: 3 HP/3 seconds
```

Reply:
Adjusted all bio calcs to match JP wiki, which has the most complete set of breakpoints (and agrees with the data you linked)